src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
Ok I now see you answered before me, I maintain my proposal though, but I'm not against yours.
 
Still I would rather not add a new flag for this, the goal here is to ease configuration ; currently you need to import an `xml` or a `php` file while `yaml` is very common (I think and always used it only).

So needing to add a flag does not seem to "keep it easy" while using `PARSE_EXCEPTION_ON_INVALID_TYPE` looks more like "keep doing as usual, just pass a constant if you want".